projects/admin/src/app/service/record-permission.service.ts

```diff
@@ -143,10 +143,11 @@ export class RecordPermissionService {
    * @param user - any
    * @param libraryPid - string
    * @param permission - any
+   * @param membership - Check record ownership
    * @returns permissions of current record
    */
-  membership(user: any, libraryPid: string, permission: any): any {
-    if (user.currentLibrary !== libraryPid) {
+  membership(user: any, libraryPid: string, permission: any, membership: boolean = true): any {
```

Comment:
you could call `service.membership(..., membership=false)` !?! Seems a little bit confused XD

Reply:
To avoid breaking compatibility, set the new parameter to True